README.md

```diff
@@ -129,7 +208,7 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+A Schema is a collection of database objects (in tables) associated with one Username. In this example, a Username would represent a single person and the objects contained within the Schema would be the wishes that have been made by that one Person.
 ```
 
```

Comment:
wouldn't call it database objects. It's the tables in a database. Objects are gonna be the entries in each table